examples/with-docker/package.json

```diff
@@ -7,12 +7,12 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@ponder/core": "latest"
```

Comment:
@0xOlias Is this a mistake?

Reply:
I think so. NGL, I haven't ran this docker example in a while so not certain it's working at all.